app/utils/replaceUnlessLoggedIn.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ComponentType } from 'react';
 
-type LoginProps = {
+export type LoginProps = {
```

Comment:
Why export this? It doesn't seem to be used

Reply:
Forgot to undo this after looking into an annoying error message concerning the `replaceUnlessLoggedIn` function